### PR TITLE
Compilation fixes + italian locale

### PR DESCRIPTION
**Pull request by hardening**
* add italian locale guess
* fixes compilation on Ubuntu 20.04

### CODE REVIEW

**Comment by hardening** [comment]
Any reviewer for this PR ?

**Comment by hardening** [comment]
is the ogon project dead ?

**Review by MartinHaimberger — COMMENTED** [review]
+1 LGTM

File: src/plugins/platforms/ogon/ogon.pro [hunk 1]
```diff
@@ -10,7 +10,7 @@ QMAKE_RPATHDIR += $$ADDITIONAL_RPATHS
 unix {
     CONFIG += link_pkgconfig
     
-    PKGCONFIG += xkbcommon glib-2.0 ogon-backend1 winpr2 fontconfig freerdp-server2 ogon-message1
+    PKGCONFIG += xkbcommon glib-2.0 ogon-backend1 winpr2 fontconfig freerdp-server2 ogon-message1 fontconfig
```
Comment (nfedera) [inline src/plugins/platforms/ogon/ogon.pro]: 2 times fontconfig ?

**Comment by hardening** [comment]
@nfedera ok fixed

**Pull request merged**